canonical-web-and-design/snapcraft.io
# Update first snap flow

## Done
- Removed code from first snap flow a/b test
- Updated "push" terminology to "upload"

## QA
- Go to https://snapcraft-io-3556.demos.haus/first-snap-2/python/macos/upload
- Check that it is a 404
- Go to https://snapcraft-io-3556.demos.haus/first-snap-3/python/macos/upload
- Check that it is a 404
- Go to https://snapcraft-io-3556.demos.haus/first-snap/python/macos/push
- Check that it redirects to https://snapcraft-io-3556.demos.haus/first-snap/python/macos/upload
- Sign in with the "Register/sign in" button
- Click open "Step 3" and check that the command line option is `snapcraft upload` and not `snapcraft push`

## Issue
Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/3552, https://github.com/canonical-web-and-design/snapcraft.io/issues/3505
